bin/webpack-helpers.js

```diff
@@ -124,6 +124,8 @@ const mainEntry = {
 	'active-filters': './assets/js/blocks/active-filters/index.js',
 	'block-error-boundary':
 		'./assets/js/base/components/block-error-boundary/style.scss',
+	'wordpress-components-style':
```

Comment:
why do we need this here?

Reply:
That's required to load `@wordpress/components` style so the `<Panel>` component looks as expected.

Reply:
> That's required to load @wordpress/components style so the <Panel> component looks as expected.

Will the [`@wordpress/base-styles`](https://www.npmjs.com/package/@wordpress/base-styles) package help with this?

Reply:
I don't think so. That package seems to include several SCSS variables used in Gutenberg styles, but in this case we want the `<Panel>` component styles, which are not included in that package.